relay-profiling/src/extract_from_transaction.rs

```diff
@@ -15,14 +15,11 @@ pub fn extract_transaction_metadata(event: &Event) -> BTreeMap<String, String> {
     if let Some(dist) = event.dist.as_str() {
         tags.insert("dist".to_owned(), dist.to_owned());
     }
-    if let Some(dist) = event.dist.value() {
```

Comment:
I feel like I wanted to extract something else but never renamed it and I forgot what it was.